diff --git a/apis/config/types.go b/apis/config/types.go
index d2496083..90a86cb3 100644
--- a/apis/config/types.go
+++ b/apis/config/types.go
@@ -16,11 +16,17 @@ limitations under the License.
 
 package config
 
+
 import (
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	schedconfig "k8s.io/kubernetes/pkg/scheduler/apis/config"
 )
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+type EnergyScoreArgs struct {
+	metav1.TypeMeta
+	WeightMultiplier float64 `json:"weightMultiplier,omitempty"`
+}
 
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
@@ -298,3 +304,4 @@ type PowerModel struct {
 	// Power = K0 + K1 * e ^(K2 * x) : where x is utilisation
 	// Idle power of node will be K0 + K1
 }
+
