diff --git a/apis/config/v1/zz_generated.defaults.go b/apis/config/v1/zz_generated.defaults.go
index b13649bd..153afbbf 100644
--- a/apis/config/v1/zz_generated.defaults.go
+++ b/apis/config/v1/zz_generated.defaults.go
@@ -30,6 +30,7 @@ import (
 // All generated defaulters are covering - they call all nested defaulters.
 func RegisterDefaults(scheme *runtime.Scheme) error {
 	scheme.AddTypeDefaultingFunc(&CoschedulingArgs{}, func(obj interface{}) { SetObjectDefaults_CoschedulingArgs(obj.(*CoschedulingArgs)) })
+	scheme.AddTypeDefaultingFunc(&EnergyScoreArgs{}, func(obj interface{}) { SetObjectDefaults_EnergyScoreArgs(obj.(*EnergyScoreArgs)) })
 	scheme.AddTypeDefaultingFunc(&LoadVariationRiskBalancingArgs{}, func(obj interface{}) {
 		SetObjectDefaults_LoadVariationRiskBalancingArgs(obj.(*LoadVariationRiskBalancingArgs))
 	})
@@ -52,6 +53,10 @@ func SetObjectDefaults_CoschedulingArgs(in *CoschedulingArgs) {
 	SetDefaults_CoschedulingArgs(in)
 }
 
+func SetObjectDefaults_EnergyScoreArgs(in *EnergyScoreArgs) {
+	SetDefaults_EnergyScoreArgs(in)
+}
+
 func SetObjectDefaults_LoadVariationRiskBalancingArgs(in *LoadVariationRiskBalancingArgs) {
 	SetDefaults_LoadVariationRiskBalancingArgs(in)
 }
