diff --git a/apis/config/v1/defaults.go b/apis/config/v1/defaults.go
index 245f2e6a..f1cecbbc 100644
--- a/apis/config/v1/defaults.go
+++ b/apis/config/v1/defaults.go
@@ -250,3 +250,9 @@ func SetDefaults_SySchedArgs(obj *SySchedArgs) {
 		obj.DefaultProfileName = &DefaultSySchedProfileName
 	}
 }
+func SetDefaults_EnergyScoreArgs(obj *EnergyScoreArgs) {
+	if obj.WeightMultiplier == 0 {
+		obj.WeightMultiplier = 1.0
+	}
+}
+
