diff --git a/cmd/scheduler/main.go b/cmd/scheduler/main.go
index 7c915be2..ff84e382 100644
--- a/cmd/scheduler/main.go
+++ b/cmd/scheduler/main.go
@@ -38,6 +38,7 @@ import (
 	"sigs.k8s.io/scheduler-plugins/pkg/trimaran/lowriskovercommitment"
 	"sigs.k8s.io/scheduler-plugins/pkg/trimaran/peaks"
 	"sigs.k8s.io/scheduler-plugins/pkg/trimaran/targetloadpacking"
+	"sigs.k8s.io/scheduler-plugins/pkg/energyscore"
 
 	// Ensure scheme package is initialized.
 	_ "sigs.k8s.io/scheduler-plugins/apis/config/scheme"
@@ -64,6 +65,8 @@ func main() {
 		// app.WithPlugin(crossnodepreemption.Name, crossnodepreemption.New),
 		app.WithPlugin(podstate.Name, podstate.New),
 		app.WithPlugin(qos.Name, qos.New),
+		app.WithPlugin(energyscore.Name, energyscore.New),
+
 	)
 
 	code := cli.Run(command)
